Gemfile-rails-dependencies

```diff
@@ -25,4 +25,15 @@ else
   gem "puma"
   gem 'activerecord-jdbcsqlite3-adapter', platforms: [:jruby]
   gem 'selenium-webdriver', require: false
+
+  # This is required for Ruby 3.1 and Rails 6.1.x as of time
+  # of writing, because in Ruby 3.1 these gems are no longer
+  # automatically included.  This issue was fixed on the Rails
+  # side in Rails 7.0.1, but is not yet fixed in the Rails 6.1.x
+  # branch.  Discussion can be found here - https://github.com/mikel/mail/pull/1439
+  if RUBY_VERSION >= '3.1' && version.split(' ').last < '7.0'
+    gem 'net-smtp', require: false
+    gem 'net-imap', require: false
+    gem 'net-pop', require: false
+  end
```

Comment:
Well, it's understandeable that this didn't help. The whole thing also needs to be replicated on the `when nil, false, ""` branch, and `when /stable$/`.
It's not needed in `when /main/`, since `main` goes for the ongoing Rails 7.1 and it's fixed there.

I intend to move this outside of `case` and remove the Rails `version` check.

Reply:
When you get it sorted feel free to use dev tools to try to remove the check